index.html

```diff
@@ -34,6 +38,141 @@ <h3>I'm a Software Engineer</h3>
         <li><a href="#"><i class="fa fa-medium" aria-hidden="true"></i></a></li>
       </ul>
     </section>
+    <section class="work-section">
+      <div class="recent-works">
+        <h1>My Recent Works</h1>
```

Comment:
- [ ] Kindly make sure you don't have more than one `h1` tag on a page as it's a bad practice and it confuses the search engine when searching up your page.😉

Reply:
fixed the changes